lib/sidekiq/ultimate/fetch.rb

```diff
@@ -45,12 +46,15 @@ def retrieve_work
         work
       end
 
-      def self.bulk_requeue(units, _options)
+      # TODO: Requeue in batch or at least using pipeline
```

Comment:
Let's log an issue for this in the repo and link here

Reply:
I'll be touching this anyway in the following PR for the 7.2 migration.

Reply:
Feel free to merge then